style/package-lock.json

```diff
@@ -1,3416 +1,1006 @@
 {
   "name": "openstreetmap-americana",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
When I ran `npm install` it complained:
```
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 
```
Looks like despite requiring npm 8.3, somehow the lockfile got changed back to version 1 which hasn't been used since npm 6.